test/org/sagebionetworks/bridge/play/controllers/AuthenticationControllerMockTest.java

```diff
@@ -401,56 +405,47 @@ private void signInExistingSession(boolean isConsented, Roles role, boolean shou
         when(authenticationService.getSession(TEST_SESSION_TOKEN)).thenReturn(session);
 
         // execute and validate
-        try {
-            Result result = controller.signInV3();
-            if (shouldThrow) {
-                fail("expected exception");
-            }
-            assertSessionInPlayResult(result);
-        } catch (ConsentRequiredException ex) {
-            if (!shouldThrow) {
-                throw ex;
-            }
-        }
+        Result result = controller.signInV3();
+        assertSessionInPlayResult(result);
         verifyCommonLoggingForSignIns();    
     }
     @Test
     public void signInExistingSession() throws Exception {
-        signInExistingSession(true, null, false);
+        signInExistingSession(true, null);
     }
 
     @Test
     public void signInExistingSessionUnconsented() throws Exception {
-        signInExistingSession(false, null, true);
+        signInExistingSession(false, null);
```

Comment:
Now that we're moving the consent check to AuthenticationService, a lot of these "unconsented" tests are now defunct.

Reply:
Removed the unnecessary unconsented tests